CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Migration to ElasticSearch 6.3
 - Blob export to LinShare
 
+### Deprecated
+- Zookeeper component. This will be removed after 3.4 release. If you have development skills, and are willing to maintain this component, please reach us.
+
+### Removed
+- Karaf OSGi implementation. It was not compiled nor tested for a year. We didn't get any feedback about that and the team doesn't have the resources nor the skills to maintain it any more.
```

Comment:
I'm not easy with 'the team', we are supposed to be a community

Reply:
fixed